CHANGELOG.md

```diff
@@ -2,9 +2,32 @@
 
 ## unreleased
 
+## 3.0.0-beta.1 / 2024-10-09
```

Comment:
Let's add the new stuff that have landed in main by now. Maybe incomplete list (and not sure if tagging all the "declare stable" items as ENHANCEMENTs is the right thing):

- [FEATURE] SD: Support K8s sidecar containers in endpoint discovery. #14929
- [ENHANCEMENT](?) Declare OTLP receiver stable. #14894
- [ENHANCEMENT](?) Declare "float literals are time durations" as stable. #15111
- [BUGFIX] Re-add accidentally removed flag storage.tsdb.allow-overlapping-compaction. #15113
- [BUGFIX] storage: Fix missing histogram copy in sampleRing. #15025
- [BUGFIX] scrape: Fix protobuf parsing of unit field. #15095
- [BUGFIX] Fixes and optimisation for created timestamp ingestion. #14965

Reply:
#14965 did not look user-visible. 
#15095 is in my text. 
#15025 is in my text.
#15113 I omitted deliberately and asked about on the issue.
#15111 was marked 'docs' so omitted from changelog.
#14894 was already in CHANGELOG so not changed.
#14929 does not appear in my git log